### PR TITLE
Update sync PRs

### PR DESCRIPTION
**Pull request by SuperQ**
Push updates to the repo sync PRs if there is already a PR open. This
allows for cumulative updates to be synced.

Signed-off-by: SuperQ <superq@gmail.com>

### CODE REVIEW

**Comment by SuperQ** [comment]
cc @LeviHarrison

**Review by LeviHarrison — APPROVED** [review]
I'm not going to pretend I'm competent in shell script 🙃 , but the logic checks out.

**Review by LeviHarrison — COMMENTED** [review]
Actually wait will this discard previous changes on the branch or error:

https://github.com/prometheus/prometheus/blob/c843314a9f5cb830d14b2e27940802ae898e176c/scripts/sync_repo_files.sh#L175

And this:

https://github.com/prometheus/prometheus/blob/c843314a9f5cb830d14b2e27940802ae898e176c/scripts/sync_repo_files.sh#L79-L80

**Review by roidelapluie — APPROVED** [review]

File: scripts/sync_repo_files.sh [hunk 1]
```diff
@@ -186,6 +188,10 @@ process_repo() {
     git add .
     git commit -s -m "${commit_msg}"
     if push_branch "${org_repo}"; then
+      if [[ "${pr_link}" != "null" ]]; then
+        echo_yellow "Pull request already opened for branch '${branch}': ${pr_link}"
```
Comment (roidelapluie) [inline scripts/sync_repo_files.sh:192]: I think the deletion of the branch would close the PR so we would need to open it again. Let's see.

**Pull request merged**